docs/Gemfile

```diff
@@ -8,7 +8,7 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", ">= 3.7.4"
+# gem "jekyll", ">= 3.7.4"
```

Comment:
Why did you leave it commented out?

Reply:
Hello All,
I have two Atlassian 1 accounts in Yahoo and the other in gmail.  The Yahoo account is buggy.  I'm just a self-employed online business person.  I am retired and can not pay any other person.  This is the problem.

Reply:
> Why did you leave it commented out?

I was kind of just leaving the template in place, commenting that line back out, and uncommenting the github pages line. I can remove it.